src/pages/client/pages/ClientAddTunnelPage/style.scss

```diff
@@ -0,0 +1,126 @@
+@use '@scssutils' as *;
+
+#client-add-tunnel-page {
+  h1 {
+    @include typography(app-title);
+    color: var(--text-body-primary);
+  }
+
+  h2 {
+    @include typography(app-body-1);
+    color: var(--text-body-primary);
+  }
+  h3 {
+    @include typography(app-side-bar);
+    color: var(--text-body-primary);
+  }
+
+  form {
+    & > * {
+      width: 100%;
+    }
+
+    .controls {
+      display: flex;
+      flex-flow: row;
+      align-items: center;
+      justify-content: center;
+      .btn {
+        width: 100%;
+        max-width: 200px;
+      }
+    }
+  }
+
+  & > header {
+    width: 100%;
+    display: flex;
+    flex-flow: row;
+    align-items: center;
+    justify-content: flex-start;
+    padding-bottom: 15px;
+
+    & > h1 {
+      text-align: left;
+    }
+  }
+
+  & > .content {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: flex-start;
+    justify-content: center;
+    row-gap: 50px;
+    column-gap: 50px;
+
+    @include media-breakpoint-up(xxl) {
+      justify-content: flex-start;
+    }
+
+    & > * {
+      width: 100%;
+      max-width: 700px;
+    }
+
+    & > .card {
+      box-sizing: border-box;
+      padding: 32px 64px;
+      & > header {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-bottom: 1rem;
```

Comment:
Don't use rems we don't enforce 1:10 ratio in this project thus those spacings will be different from design.